W-10703959: Create What's New article for Studio 7.12 (#304)
mulesoft/docs-studio: modules/ROOT/pages/whats-new-in-studio.adoc

= What’s New in Studio

> Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and support for multi-factor authentication (MFA) to Anypoint Platform.

Reviewer: There seems to be a "extra" "support" verb in this sentence:
support for comprehensive publication of assets to Exchange and **_support_** for multi-factor authentication (MFA) to Anypoint Platform

> Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and support for multi-factor authentication (MFA) to Anypoint Platform.

Reviewer: ```suggestion
Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and for multi-factor authentication (MFA) to Anypoint Platform.
```

> This version also improves the metadata edition and Design Center import experiences and adds a feature to comment and uncomment objects from the canvas.

Reviewer 2: ```suggestion
This version also improves the metadata edition and Design Center import experiences and adds a feature that enables you to comment and uncomment objects directly from the canvas.
```

== Comprehensive Publication to Exchange

> When publishing an asset to Exchange, you can now add more information such as tags, categories, name, description, and documentation to your asset directly from Studio.

Reviewer 2: ```suggestion
When publishing an asset to Exchange, you can now add more information (such as tags, categories, names, descriptions, and documentation) to your assets directly from Studio.
```
I added the parens just to lessen the distraction between "add information" and "to your assets."

== MFA Support

> You can now log in to Anypoint Platform using multi-factor authentication (MFA) from Studio's built-in browser.

Reviewer: Remove possessive case in objects. Replace by "The Studio built-in browser".

> == Comment and Uncomment Objects from the Canvas

Reviewer 2: ```suggestion
== Comment and Uncomment Objects Directly from the Canvas
```
I kept struggling with whether the user was taking action *on* the canvas or *from* the canvas, but I chose to stick with your original wording.

You can now comment and uncomment flows and components directly from the canvas:

> . Right-click the flow or component that you want to comment.

Reviewer 2: ```suggestion
. Right-click the flow or component that you want to comment or uncomment.
```

. Select *Toggle Comment*.
+

> image::toggle-comment.png[]

Reviewer: Add ALT text to image.

== Improved Metadata Edition Experience

Studio 7.12 improves the overall performance of the metadata edition experience by reducing background processing and by asynchronously propagating the metadata through the Flow Reference component.

> 

Reviewer: There are two blank lines between these two paragraphs. Is this intended?

== Improved Design Center Import Experience

> Studio 7.12 improves its import from Design Center experience by loading all REST APIs to import from Design Center simultaneously enabling you to search your desired API specification much faster.

Reviewer 2: ```suggestion
Studio 7.12 improves its import from Design Center experience by loading all REST APIs to import from Design Center simultaneously, enabling you to search your desired API specification much faster.
```
